wss-agent-hash-calculator/src/main/java/org/whitesource/agent/hash/HashCalculator.java

```diff
@@ -329,6 +329,25 @@ public String calculateSha1ByNameVersionArchitectureAndType(String name, String
         return calculateByteArraySHA1(sha1ToCalc.getBytes(StandardCharsets.UTF_8));
     }
 
+    /**
+     * This is meant for the additionalSha1 of yocto.
+     * <recipeName>_<recipeVersion>_<recipeRevision>_<layer>_<yoctoTag>_<language>
+     * @param name of recipe
+     * @param version of recipe
+     * @param revision of recipe
+     * @param layer of recipe
+     * @param tag of yocto distribution-version
+     * @param type YOCTO
+     * @return Calculated SHA-1 of  recipeName_recipeVersion_recipeRevision_layer_yoctoTag_"YOCTO"
```

Comment:
This method is specialized for Yocto.
Either make it more generic and remove references to yocto, or remove the DependencyType parameter and make the "YOCTO" hard coded.